Makefile

```diff
@@ -36,6 +36,8 @@ release-binary:
 revendor:
 	@glide up -v
 	@glide-vc --use-lock-file --no-tests --only-code
+	@CGO_ENABLED=1 go get github.com/coreos/license-bill-of-materials
```

Comment:
Can we pin this instead of running go get?

Reply:
Pinned to a specific version instead of grabbing master every time. Either by vendoring to tool into Dex (what we do with the go proto generator) or by grabbing a release binary at a specific tag.

Reply:
Ok. There are no release binaries, so I'll figure out how to vendor it.